config/jobs/image-pushing/k8s-staging-kubetest2.yaml

```diff
@@ -0,0 +1,27 @@
+postsubmits:
+  kubernetes-sigs/kubetest2:
+  - name: post-kubetest2-push-binaries
+    cluster: k8s-infra-prow-build-trusted
+    annotations:
+      testgrid-dashboards: sig-testing-kubetest2
+      testgrid-alert-email: kubernetes-engprod+alerts@google.com
```

Comment:
sig-testing alerts/failure email should be here (instead?). Don't remember what it is off the top of my head

Reply:
kubernetes-sig-testing-alerts@googlegroups.com ?

Reply:
done